### PR TITLE
Investigate ci test failures

### LINKED ISSUE #726: Testing audit server issue
Occasionally we get the follow error when running tests:

```sh
  1) Auditing Integration Tests TCP Audit Server should send TCP audit messages and save (valid):

      Uncaught AssertionError: expected 1 to be 2
      + expected - actual

      -1
      +2

      at Assertion.fail (node_modules/should/lib/assertion.js:92:17)
      at Assertion.Object.defineProperty.value (node_modules/should/lib/assertion.js:164:19)
      at Query.<anonymous> (test/integration/auditingIntegrationTests.coffee:134:37)
      at node_modules/mongoose/node_modules/kareem/index.js:177:19
      at node_modules/mongoose/node_modules/kareem/index.js:109:16



Warning:  Use --force to continue.

Aborted due to warnings.
npm ERR! Test failed.  See above for more details.
```

### PR DESCRIPTION
**Pull request by napiergit**
I think this fixes #726, #727 and #728.  @rcrichton or @hnnesv can you please review.

### CODE REVIEW

**Comment by hnnesv** [comment]
👍 LGTM

File: test/integration/mediatorAPITests.coffee [hunk 1]
```diff
@@ -35,7 +35,7 @@ describe "API Integration Tests", ->
             type: 'http'
           }
         ]
-      ,
+      }, {
```
Comment (rcrichton) [inline test/integration/mediatorAPITests.coffee]: These aren't necessary because of coffee script. Where you noticing issues with them?

**Comment by rcrichton** [comment]
@napiergit this looks good to me. Are you able to repeat the build on travis a number off times to see if we get any failures? Has it been pretty stable on travis from what you have seen?

**Comment by rcrichton** [comment]
Awesome, then this is a :+1: from me

**Pull request merged**